bin/bump-version.rb

```diff
@@ -7,6 +7,19 @@
 end
 component = ARGV[0].to_sym
 
+unless `which gh` && $?.success?
```

Comment:
Should we also check if jq is installed? I noticed it was used further down in the script

Reply:
Looks like this isn't needed as it's bundled in `gh` somehow

Reply:
Sweet